### PR TITLE
ENCD-4281-fix-audit-text-message

### PR DESCRIPTION
**Pull request by zoldello**
…age box notification

### CODE REVIEW

File: src/encoded/static/components/auditmatrix.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { svgIcon } from '../libs/svg-icons';
 import * as globals from './globals';
 import { BrowserSelector } from './objectutils';
 import { BatchDownload, FacetList, TextFilter } from './search';
+import pluralize from 'pluralize';
```
[the remark this answers is not in the document]
Comment (zoldello) [reply src/encoded/static/components/auditmatrix.js]: @forresttanaka Thanks for the catch. It has been addressed.

File: src/encoded/static/components/matrix.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { svgIcon } from '../libs/svg-icons';
 import * as globals from './globals';
 import { BrowserSelector } from './objectutils';
 import { BatchDownload, FacetList, TextFilter } from './search';
+import pluralize from 'pluralize';
```
Comment (forresttanaka) [inline src/encoded/static/components/matrix.js]: This still needs to be put above underscore.

Comment (zoldello) [reply src/encoded/static/components/matrix.js]: If I am not mistaken, the above is old and was fixed yesterday. However, I did not update auditmatrix.js. That has been fixed.

**Comment by zoldello** [comment]
@caseylitton I rebased this branch on dev.

File: src/encoded/static/components/auditmatrix.js [hunk 3]
```diff
@@ -233,7 +237,7 @@ class AuditMatrix extends React.Component {
                                         <div>
                                             <h3 style={{ marginTop: 0 }}>{context.title}</h3>
                                             <div>
-                                                <p>Click or enter search terms to filter the experiments included in the matrix.</p>
+                                                <p>Click or enter search terms to filter the {type} included in the matrix.</p>
```
Comment (caseylitton) [inline src/encoded/static/components/auditmatrix.js]: This is a lot of work for what seems like a one line, one word change.  Don't we just need to change 'experiments' to 'annotations' in the auditmatrix?

Comment (caseylitton) [reply src/encoded/static/components/auditmatrix.js]: Ignore per discussion, I miss understood the changes.

**Pull request merged**